### PR TITLE
Remove packages from the tests folder

### PR DESCRIPTION
**Pull request by honzajavorek**
Close https://github.com/pyvec/python.cz/issues/301

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [scripts]
-test = "pytest ./tests --cov=pythoncz --pylama"
+test = "python -m pytest ./tests --cov=pythoncz --pylama"
```
Comment (honzajavorek) [inline Pipfile]: This adds current directory to the Python path and makes imports working

File: tests/models/github_test.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 from pythoncz import app
 from pythoncz.models import github as github_module
-from tests.models import github_test_fixtures as fixtures
+import github_test_fixtures as fixtures
```
Comment (honzajavorek) [inline tests/models/github_test.py]: I just tried this and it works 😄 Ideally the tests would be probably simpler, maybe not needing the file at all, IDK

File: Pipfile (hunk 1 above)
Comment (messa) [reply Pipfile]: Seems strange. So pipenv install does not install pythoncz?

Comment (hroncok) [reply Pipfile]: There is no more pythoncz to install.

Comment (hroncok) [reply Pipfile]: Also, documented at https://docs.pytest.org/en/latest/usage.html#calling-pytest-through-python-m-pytest

**Comment by honzajavorek** [comment]
Yup, its a classic problem with pipenv I’m facing on all projects. Once the
project isnt a setup.py package, this whole import stuff gets hairy.

Sometimes I miss Node’s plain import system, where you say I want
“../something.js” and it just gets imported, without any WHATEVERPATH magic.

**Comment by messa** [comment]
> Yup, its a classic problem with pipenv I’m facing on all projects. Once the project isnt a setup.py package, this whole import stuff gets hairy.

How do you run it, then? Only via pipenv run? Pipenv discourages the use of setup.py?

**Comment by honzajavorek** [comment]
setup.py doesnt make sense, pipenv is like requirements.txt - which, I
guess, has to have simliar problems? 🤔

And you can run it how you want. “pipenv run” is just a way to alias your
commands so you don’t have to remember them/document them and they have a
uniform interface. Like npm scripts, but simpler. Or Makefile, but
primitive.

The sad reality of pipenv scripts is they do NOT do anything special, like
adding cwd to pythonpath.

**Comment by honzajavorek** [comment]
Thanks!

**Pull request merged**